### ISSUE TITLE
disabling (only) some but not all of the 3 Windows Services? 

### ISSUE THREAD

**Issue by pstein** [WereDev/Wu10Man#35]
On tab "Windows Services" of Win10man I can disable 3 different Update Services.

I cannot find a scenario where users should only disable 1 or 2 of them.

So shouldn't it be better to merge them into ONE switch for all the 3 Services?

Furthermore I think that disabling 1,2 or 3 of the services would automatically imply a pause and infinite defer of all updates on tab 2 "Pause Updates" of Win10man.

In other words: If I disable 1, 2 or3 of the services on tab 1 I am not able to receive "Quality updates" any more.

Can someone confirm this?

Thank you
Peter

**Comment by JonnyTech** [comment]
I sometimes need to enable just one for installing microsoft drivers or runtimes without opening the floodgates to all other unwanted updates.

**Comment by englishextra** [comment]
@pstein 

- When you update Mail app you need to enable updates service and trusted installer and not medicSvc which is the most evel one, also you need to unblock ms servers in hosts file on a second tab
- if you update Adobe Acrobat you need to enable only trusted installer

Merging all three in one in the first tab will turn the tweaker into garbage and a user who understands what's going on will have to manually do the work.

@WereDev Please return the tab with blocking servers in host file - it is really important for thos who understand how a forced update process is working.

Thanks

**Comment by WereDev** [comment]
@pstein 
The latest version of Wu10Man also has a disable/enable all services button near the bottom so that you can do it in one click, or enable/disable individual services as you might need.

The Pause functionality on the 3rd tab is actually part of newer Windows Update functionality provided by Microsoft.  The values allow a user to kind of do what I'm setting up by disabling services, but it's a wholly different set of functionality.  I added a screen here to get around the max values Microsoft has in place in a few spots, but is otherwise just a screen to support that functionality and is not functionality that I built out.  I think it might be awkward if disabling the services also set those values, but it's something I can consider.

@englishextra
The hosts file editor is still there.  It's in the Legacy section now along with the old Group Policy Editor stuff.  If Hosts really is that popular, I can move it back to the main screen in a future update.
